### PR TITLE
Rebase images on microsoft/dotnet

### PR DESCRIPTION
**Pull request by natemcmaster**
/cc @glennc @troydai

### CODE REVIEW

File: 1.1.0-preview1/jessie/build/Dockerfile [hunk 1]
```diff
@@ -1,44 +1,11 @@
...
 ENV ASPNETCORE_URLS http://+:80
 
 # set env var for packages cache
 ENV DOTNET_HOSTING_OPTIMIZATION_CACHE /packagescache
 
-# set env var for skipping .NET Core SDK first time experience
-ENV DOTNET_SKIP_FIRST_TIME_EXPERIENCE 1
-
-# set env var for minimizing package restore footprint
-ENV NUGET_XMLDOC_MODE Skip
```
Comment (troydai) [inline 1.1.0-preview1/jessie/build/Dockerfile]: We need this.

File: 1.1.0-preview1/jessie/build/Dockerfile [hunk 2]
```diff
@@ -1,44 +1,11 @@
...
+FROM microsoft/dotnet:1.0.0-preview2.1-sdk
 
 # set up network
 ENV ASPNETCORE_URLS http://+:80
 
 # set env var for packages cache
 ENV DOTNET_HOSTING_OPTIMIZATION_CACHE /packagescache
 
-# set env var for skipping .NET Core SDK first time experience
-ENV DOTNET_SKIP_FIRST_TIME_EXPERIENCE 1
```
Comment (troydai) [inline 1.1.0-preview1/jessie/build/Dockerfile]: Can you confirm if it is set in the base image?

Comment (natemcmaster) [inline 1.1.0-preview1/jessie/build/Dockerfile]: They do a dotnet restore in the base image, which prepopulates the nuget cache. https://github.com/dotnet/dotnet-docker/blob/master/1.0.0-preview2/debian/Dockerfile#L39

Comment (troydai) [inline 1.1.0-preview1/jessie/build/Dockerfile]: Good. But I'm curious how much does this increase the size the image.

**Pull request merged**